qml/pages/All.qml

```diff
@@ -313,7 +319,9 @@ Page {
 
                         Label {
                             id: project
-                            text: "[" + netDur.toString().toHHMM() + "]  " + model.projectName + "  " + model.taskName
+                            text: "[" + netDur.toString().toHHMM() + "]  "
+                                  + model.projectName
+                                  + (model.taskName !== '' ? " / " + model.taskName : "")
```

Comment:
This makes the task name appear twice. It's displayed already below as its own `Label`

Reply:
I decided to delete the other one in https://github.com/olpeh/wht/commit/269cc56685ef80da83730af63f182f43803b3f76